jerry-core/jerry.cpp

```diff
@@ -1067,6 +1125,7 @@ jerry_api_set_object_native_handle (jerry_api_object_t *object_p, /**< object to
                                                            ECMA_INTERNAL_PROPERTY_FREE_CALLBACK);
     if (prop_p != NULL)
     {
+      ecma_free_external_pointer_in_property (prop_p);
       ecma_delete_property (object_p, prop_p);
```

Comment:
The free operation is also performed in `ecma_free_property` that is called from `ecma_delete_property`, so it is not necessary to call `ecma_free_external_pointer_in_property` here.